distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -103,32 +113,38 @@ Prior to installing ${branding}, ensure the system time is accurate to prevent f
 To install the ${branding} distribution zip, perform the following:
 
 . Download the ${branding} {download-url}[zip file].
-. After the prerequisites have been met, as a root user if for *NIX, change the current directory to the desired install location.
+. After the prerequisites have been met, create or change to the current directory to the desired install location.
```

Comment:
Yeah, that prerequisite section.  I made it a link. Revised the second part of the sentence as well.